forgerock-core/src/main/java/org/forgerock/android/auth/NetworkConfig.java

```diff
@@ -1,31 +1,30 @@
 /*
```

Comment:
i want to convert this to kotlin at some point in a different story .. but since i modified the OKHttpClientProvider to kotlin , i have to delombak this class